src/about-this-guide.md

```diff
@@ -3,33 +3,41 @@
 This guide is meant to help document how rustc – the Rust compiler – works,
 as well as to help new contributors get involved in rustc development.
 
-There are seven parts to this guide:
+There are nine parts to this guide:
 
-1. [Building `rustc`][p1]:
+1. [Building and debugging `rustc`][p1]:
    Contains information that should be useful no matter how you are contributing,
    about building, debugging, profiling, etc.
-2. [Contributing to `rustc`][p2]:
+2. [Contributing to Rust][p2]:
```

Comment:
numbering them all `1.` avoid having to care how many items there, and keeping such enumeration current
```suggestion
1. [Contributing to Rust][p2]:
```

Reply:
Ack, will do shortly.

Reply:
Done.